addons/website/views/snippets/s_alert.xml

```diff
@@ -12,19 +12,24 @@
 
 <template id="s_alert_options" inherit_id="website.snippet_options">
     <xpath expr="." position="inside">
-        <div data-selector=".s_alert">
+        <div data-selector=".s_alert" data-js="Alert">
```

Comment:
Why ?
EDIT: mmh in fact I think it is just your commit which is not split correctly, the Alert option should be in here
EDIT2: mmh in fact I cannot find that Alert option anywhere ?

Reply:
Weird, I find the code back through `reflog`, you might have another look, the options JS is now there.